docs/RTL.md

```diff
@@ -0,0 +1,41 @@
+# Using RTL
+
+Right-to-left (also known as RTL) is a form of writing so that the direction starts from the right and continues to the left.
+
+The components of `wix-ui-tpa` are RTL-compatible and support that built-in, meaning that all the structure of the component changes from right to the left as well.
+
+## How to Use
+
+In order to apply RTL correctly for the library we'll need to use two things:
+
+1. Using `TPAComponentsProvider` while enabling the `rtl` option
+2. Setting the `dir` attribute as `rtl` on the root element
```

Comment:
Also getting the direction from the `body` is ok, so I guess we can say that consumers just need to ensure the components have an ancestor with the correct direction.
I mean, adding the `dir` attribute to the root of the app will do, but they might get it also from the viewer.
What do you think?

Reply:
Good point. 

I'll change the second bullet to something like:
"Making sure the `dir` attribute is already set as `rtl`. Notice that the viewer might set the correct direction by itself on the `body` or on an ancestor of the component. If not, we can always set it on the root of the component by ourselves."

I'll keep the example as:
```
const MyApp = () => <div dir="rtl">Hello World!</div>;
```

But will mention in the comment above that the viewer also can set it.

WDYT?

Reply:
Excellent 🙌